riscv-asm.md

```diff
@@ -214,7 +214,150 @@ ATTRIBUTE_VALUE := <string>
 
 #### `rvc`/`norvc`
 
-Enable/disable the C-extension for the following code region.
+This option will be deprecated soon after `.option arch` has been widely
+implemented on main stream open source toolchains.
+
+Enable/disable the C-extension for the following code region. This option is
+equivalent to `.option arch, +c`/`.option arch, -c`, but widely support by
+older toolchain versions.
+
+Alternative style:
+
+```
+.option push
+.option +c   # Alternative of .option rvc
+.option pop
+
+.option push
+.option -c   # Alternative of .option norvc
+.option pop
+```
+
+NOTE: .option rvc might set the ELF flag EF_RISCV_RVC in some toolchains. That
+might cause the linker to compress instructions in code regions where that was
+not intended.
+
+NOTE: There is a difference between `.option rvc`/`.option norvc` and
+`.option arch +c`/`.option arch -c`. The latter won't set EF_RISCV_RVC in the
```

Comment:
`.option arch, +c`

Better not to allow `.option .arch +c` (space-separated).

Reply:
Updated, that's typo, I am not intend to support space-separated form